websites/A/Animevost/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "denfake2",
+		"id": "870352885089730561"
+	},
+	"service": "Animevost",
+	"description": {
+		"en": "Base #1 on watching anime online for free.",
+		"ru": "База №1 по просмотру аниме онлайн бесплатно."
+	},
+	"url": "v2.vost.pw",
```

Comment:
This URL is not the main URL of the service, please use this instead: https://animevost.org/